ocaml/libs/tracing/tracing.mli

```diff
@@ -40,6 +40,16 @@ module Status : sig
   type t
 end
 
+module Attributes : sig
+  type 'a t
+
+  val fold : (string -> 'a -> 'b -> 'b) -> 'a t -> 'b -> 'b
```

Comment:
This smells funny. We have a new abstract type `'a t` that has no way to create or observe it.